blog/content/post/2021-02-23-OSF_vPub_01.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "OSF vPub (vBeer) - what a fun event!"
+abstract: "We've just had our first OSF vPub gathering, with lots of
+          fun & really interesting talk about the open-source
+          firmware/hardware with 50 masters from all over the world!"
+cover: /covers/OSF_vPub.png
+author: mike.banon
+layout: post
+published: true
+date: 2021-02-23
+archives: "2021"
+
+tags:
+  - Conference
+  - Event
+  - vPub
+  - Firmware
+  - Open-source
+categories:
+  - Firmware
+  - Miscellaneous
+
+---
+
+Our meeting has been attended by ~50 firmware masters from all over the world!
+
+Together with our prominent 3mdeb hosts - Piotr Król and Michał Żygowski - it
+was an honour to be visited by such rockstars as Thierry Laurion from [Insurgo][1],
+Tom Hiller aka Thrilleratplay, Daniel Maslowski aka CyReVolt, Piotr Kubaj,
+Andrew Cooper, Felix Singer, David Hendricks, and many others whom I forgot to
+mention (sorry!) - or who preferred to stay anonymous and participate via chat.
+
+We've also had some surprise guests: despite being the relative newcomers to our
+cosy community, they have managed to ignite quite an interesting discussion!
+Alexey Vazhnov described how tricky could it be to get merged the coreboot docs,
+and we've thought about how to improve this workflow & make this process easier.
+
+In addition to the fresh coreboot forks like heads & oreboot and [new nice GUIs][2],
+we've tried to figure out how to promote the coreboot for its official support
+by the PC manufacturers - at least for those with new AMD-based motherboards!
+And now we know, how we can - even if not "bring a coreboot to every house" -
+but to significantly spread its' popularity by wisely doing our joint activism.
+
+Also, we've talked a lot about the open hardware: for example, two interesting
+boards from Crowdsupply - [ULX3S][3] and [Qomu][4] - have a small FPGA for which
+an opensource toolchain is available. Also, we've explored the firmware freedom
+in a "world without x86", [Talos II][5] workstation with a CPU(s) of POWER9 arch,
+could POWER10 become as libre as POWER9 regarding the firmware, and much more!
```

Comment:
@mikebdp2 D-DIMM topic was big part of that, so it would be great to have link here.

Reply:
@pietrushnic Done